tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/internal/07-liferay-code-upgrade-plan.markdown

```diff
@@ -63,7 +63,7 @@ Here are the steps for upgrading your development environment and custom plugins
         <div class="description">Upgrade your build dependencies.</div>
     </li>
     <ol>
-        <li>
+        <li commandId="update_repository_url">
```

Comment:
Can you update the markdown over on my branch here?  https://github.com/gamerson/liferay-docs/tree/upgradePlanner as well?